src/pages/NewTodoPage.tsx

```diff
@@ -1,7 +1,60 @@
-import React from "react";
+import React, { useContext } from "react";
+import InputDetail from "../components/InputDetail";
+import { Container } from "@mui/system";
+import GlobalContext from "../contexts/global";
+import { useNavigate, useParams } from "react-router-dom";
 
 const NewTodoPage = () => {
-  return <div>NewTodoPage</div>;
+  const { state, setState } = useContext(GlobalContext);
```

Comment:
this is a lot of repeat from the `AddPage` component where we used this same kinda pattern to add a category - i'm repurposing that here to add a `task`